akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -278,6 +285,7 @@ object ClusterEvent {
         case (_, newMember :: oldMember :: Nil) if newMember.status != oldMember.status ⇒ newMember
       }
       val memberEvents = (newMembers ++ changedMembers) collect {
+        case m if m.status == Joining  ⇒ MemberJoined(m)
         case m if m.status == WeaklyUp ⇒ MemberWeaklyUp(m)
         case m if m.status == Up       ⇒ MemberUp(m)
         case m if m.status == Exiting  ⇒ MemberExited(m)
```

Comment:
Not sure. Is the difference between `Leaving` and `Exiting` important for users? I always thought of that more as an internal detail.